editor/src/components/canvas/canvas-strategies/canvas-strategies.spec.browser2.tsx

```diff
@@ -161,13 +166,16 @@ describe('Strategy Fitness', () => {
       RegisteredCanvasStrategies,
       pickCanvasStateFromEditorState(renderResult.getEditorState().editor),
       interactionSession,
-      baseStrategyState(renderResult.getEditorState().editor.jsxMetadata),
+      baseStrategyState(
+        renderResult.getEditorState().editor.jsxMetadata,
+        renderResult.getEditorState().editor.allElementProps,
+      ),
       null,
     )
 
     expect(canvasStrategy.strategy?.strategy.id).toEqual('ESCAPE_HATCH_STRATEGY')
   })
-  it('fits Escape Hatch Strategy when dragging a flex element without siblings', async () => {
+  xit('fits Escape Hatch Strategy when dragging a flex element without siblings', async () => {
```

Comment:
Should this test be removed? Or updated?

Reply:
this should be left disabled until we decide what we want with the escape hatch interaction. I made a todo in the paper doc!

Reply:
ugh but `fits Escape Hatch Strategy when dragging a flow element` is going to break because I did not `xit` that. 
what should I do for this PR?
1. leave the escape hatch disabled as it is, and xit the `fits Escape Hatch Strategy when dragging a flow element` test
2. re-enable the escape hatch for only flow elements, where it still makes sense, because the 4 new reparent strategies don't deal with flow anyways?

Reply:
Don't mind much since we'll need to get to flow very quickly after this (meaning: once we're happy with Flex). I'd say, for this PR whatever is less distracting.

Reply:
thanks! I'll just disable it for now then, maybe that will also make it clearer that we don't support flow yet.